lib/Optimizer/GraphOptimizer.cpp

```diff
@@ -855,6 +855,126 @@ static void optimizeBatchNorm(Function *F) {
   } // For all nodes in the graph.
 }
 
+// \returns true if all dimensions of the input tensors are the same except
+// for the provided dimension, otherwise return false.
+static bool
+verifyInputsForConcat(const llvm::SmallVectorImpl<NodeValue> &inputs,
```

Comment:
Please use ArrayRef instead of SmallVectorImpl here.

Reply:
fixed